desc/grid.py

```diff
@@ -1596,3 +1572,81 @@ def find_least_rational_surfaces(
     io = find_most_distant(io_rat, n, a, b, tol=atol, **kwargs)
     rho = _find_rho(iota, io, tol=atol)
     return rho, io
+
+
+def meshgrid_inverse_idx(a_size, b_size, c_size):
```

Comment:
would it make sense to maybe make a new constructor method for the `Grid` class like `Grid.create_meshgrid(a,b,c)`?

Reply:
Yes!